__tests__/database.test.js

```diff
@@ -0,0 +1,32 @@
+const testData = require('../db/data/test-data/index.js');
+const seed = require("../db/seeds/seed.js");
+const db = require("../db/connection.js");
+const request = require("supertest");
+const app = require("../app.js");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("GET topics", () => {
+  test("Request from /api/topics returns array of topics from the database", () => {
+    return request(app)
+    .get('/api/topics')
+    .expect(200)
+    .then(({body}) => {
+        expect(body).toEqual(testData.topicData)
+    })
+  });
+  test("Request from /api/topics! returns 404 not found", () => {
```

Comment:
This can be a general 404-path not found test - it's not specific to the topics endpoint! :D